coalib/settings/Setting.py

```diff
@@ -112,20 +136,23 @@ def __init__(self,
         self.key = key
         self.origin = str(origin)
 
-    def __path__(self, origin=None):
+    def __path__(self, origin=None, glob_escape_origin=False):
         """
         Determines the path of this setting.
 
         Note: You can also use this function on strings, in that case the
         origin argument will be taken in every case.
 
-        :param origin:      the origin file to take if no origin is specified
-                            for the given setting. If you want to provide a
-                            directory, make sure it ends with a directory
-                            seperator.
-        :return:            An absolute path.
-        :raises ValueError: If no origin is specified in the setting nor the
-                            given origin parameter.
+        :param origin:             the origin file to take if no origin is
```

Comment:
Sorry, only the capital

On Thu, Mar 17, 2016 at 7:15 PM, Adhityaa Chandrasekar <
notifications@github.com> wrote:

> In coalib/settings/Setting.py
> <https://github.com/coala-analyzer/coala/pull/1644#discussion_r56505178>:
>
> >          """
> >          Determines the path of this setting.
> >
> >          Note: You can also use this function on strings, in that case the
> >          origin argument will be taken in every case.
> >
> > -        :param origin:      the origin file to take if no origin is specified
> > -                            for the given setting. If you want to provide a
> > -                            directory, make sure it ends with a directory
> > -                            seperator.
> > -        :return:            An absolute path.
> > -        :raises ValueError: If no origin is specified in the setting nor the
> > -                            given origin parameter.
> > +        :param origin:             the origin file to take if no origin is
>
> What period? It's there at the end.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1644/files/5f92726abba97e9fc0733ca1d8b86bde85b0f011..c98855d1da4f97cf231df0e1043b8de04a4069c1#r56505178>
>